python/cornserve/task/builtins/llm.py

```diff
@@ -269,6 +269,87 @@ def invoke(self, task_input: OpenAIChatCompletionRequest) -> Stream[OpenAIChatCo
         return self.llm.invoke(task_input)
 
 
+class MLLMEmbeddingTask(Task[OpenAIChatCompletionRequest, LLMEmbeddingResponse]):
```

Comment:
So the source code of this composite task is nearly (or entirely) identical to `MLLMTask` but you had to create this because our current task abstraction can only support one output type at a time, right?

It would be nice if you could note this in a comment so that we can collect all the problems of the task abstraction later and fix.